support-frontend/assets/pages/[countryGroupId]/components/formSection.tsx

```diff
@@ -0,0 +1,25 @@
+import { css } from '@emotion/react';
+import { from, space } from '@guardian/source/foundations';
+import type { ReactNode } from 'react';
+
+const fieldset = css`
+	position: relative;
+
+	& > *:not(:first-of-type) {
+		margin-top: ${space[3]}px;
+	}
+
+	${from.tablet} {
+		& > *:not(:first-of-type) {
+			margin-top: ${space[4]}px;
+		}
+	}
+`;
+
+type FormSectionProps = {
+	children: ReactNode;
+};
+
+export function FormSection({ children }: FormSectionProps) {
```

Comment:
Feels like a potentially unnecessary abstraction but it's quite nice to [colocate component with the styles](https://emotion.sh/docs/best-practices#colocate-styles-with-components)

Reply:
If we're going to be having more than 1 for, this makes sense. I might consider having a `form.tsx` that just contains it all - but that's a stylistic thing.

[I quite like the pattern they have here](https://emotion.sh/docs/best-practices#consider-defining-styles-outside-your-components), which includes co-location of components that are coupled (a `FormSection` would never live outside a form) and keeping styles related to their component.

Reply:
Could be a good option - there are some other shared bits that don't feel deserving of an entire file. Let's come back to this in the next PR